config/jsdoc/api/plugins/doc.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Define an @doc tag
+ */
+var conf = env.conf.stability;
+var defaultLevels = ["deprecated","experimental","unstable","stable","frozen","locked"];
```

Comment:
there is the stable/unstable distinction though. Is that not what this is handling?